picturae_importer.py

```diff
@@ -395,22 +463,25 @@ def generate_taxon_fields(self, index, taxon):
 
         author_insert = self.author
 
-        if rank_name != self.family_name:
+        if rank_name != self.family_name and rank_name != self.genus:
             tree_item_id, rank_id = self.taxon_assign_defitem(taxon_string=rank_name)
+        elif rank_name == self.genus:
+            rank_id = 180
+            tree_item_id = 12
         else:
             rank_id = 140
             tree_item_id = 11
 
```

Comment:
skipping author insert if TNRS score not high enough for full name, will still match author for higher parent taxa.